bin/watch

```diff
@@ -69,7 +74,6 @@ let errors = 0;
 function compilationStarted() {
     if (mochap) {
         mochap.kill("SIGINT");
-        console.log();
```

Comment:
Some mocha reporters write a text without moving on a new line, like printing “ test1:” and after a while appending status “ test1: x”. If this doesn’t add a line here output of these reporters gets mixed.

Reply:
I will check and reintroduce it if necessary.

Reply:
I have tried the watcher using the available package under test/it/fixtures/packages/watcher with various reporters, and the output seemed fine for all of them (nyan, spec, json, dot).